tech-far-hub/src/pages/index.tsx

```diff
@@ -1,97 +1,69 @@
 import * as React from "react";
 import { HeadFC, Link, PageProps, graphql } from "gatsby";
-import { StaticImage } from "gatsby-plugin-image";
 import SiteLayout from "../components/site-layout";
 import { Grid } from "@trussworks/react-uswds";
 import { Initiative } from "../components/initiative";
+import { GatsbyImage, getImage } from "gatsby-plugin-image";
 
-const IndexPage: React.FC<PageProps<Queries.HomePageInitiativesQuery>> = ({
+const IndexPage: React.FC<PageProps<Queries.HomePagePromosQuery>> = ({
   data,
 }: {
-  data: Queries.HomePageInitiativesQuery;
+  data: Queries.HomePagePromosQuery;
   children: undefined;
 }) => {
-  const initiatives = data.allMdx.nodes.map((node) => {
+  const initiatives = data.initiatives.nodes.map((node) => {
     if (
       node.frontmatter &&
-      node.frontmatter.promo_description &&
+      node.frontmatter.link &&
       node.frontmatter.heading &&
-      node.frontmatter.slug &&
       node.frontmatter.media_image &&
-      node.frontmatter.media_alt &&
-      node.parent &&
-      "relativeDirectory" in node.parent
+      node.html
     ) {
-      const pagePath = `/${node.parent.relativeDirectory}/${node.frontmatter.slug}`;
       return (
-        <Initiative heading={node.frontmatter.heading} destination={pagePath} media={node.frontmatter.media_image}>
-          {node.frontmatter.promo_description}
+        <Initiative
+          heading={node.frontmatter.heading}
+          destination={node.frontmatter.link}
+          media={node.frontmatter.media_image}
+          media_alt={node.frontmatter.media_alt_text || "An image"}
+        >
+          <div dangerouslySetInnerHTML={{ __html: node.html }} />
         </Initiative>
       );
     }
   });
+  let caseStudyPromo = null;
+  console.log("Case Study", JSON.stringify(data.caseStudy));
```

Comment:
keep console.log()?

Reply:
I need to add a pre-push hook to stop me from leaving them in.